meteor-app/imports/api/assessments/test/assessment.test.js

```diff
@@ -0,0 +1,137 @@
+import { resetDatabase } from '/imports/test/util-test'
+import { expect } from 'chai'
+import faker from 'faker'
+import { Factory } from 'meteor/dburles:factory'
+
+import Assessment, { STATUS } from '/imports/api/assessments/assessment'
+
+// TODO: More examples of badAssessment form data structure
+const badAssessment = [{
+  customerDetails: {
+    name: 'test',
+    phone: 'test',
+    refurbishment: false,
+  },
+  bikeDetails: {
+    make: 'Dahon',
+    color: 'Red',
+    bikeValue: 'Five',
+  },
+  services: {
+    serviceItem: [
+      {},
+      {
+        name: 'Fix tyre',
+        price: '500',
+      }
+    ],
+    totalServiceCost: '2000',
+  },
+  parts: {
+    partsItem: [
+      {},
+      {
+        name: 'Handle Bar',
+        price: '2000',
+      },
+    ],
+    totalPartsCost: '2000',
+  },
+  additionalFees: '2000',
+  totalCost: '2000',
+  dropoffDate: new Date(),
+  pickupDate: new Date(),
+  urgent: 'true',
+  assessor: 'James',
+  mechanic: 'Mike',
+  comment: '',
+  temporaryBike: 'false',
+  status: 'Completed',
+  search: 'abc'
+}]
+
```

Comment:
You can do this much more simply by using the factory function: this will give you a well formed assessment:

`const a = Factory.build('assessment')`

and this will give you one with the status set to 99

`const a = Factory.build('assessment',{ status: 99})`

So this way it's pretty easy to create bad conditions by using a known good object, and just making one field bad at a time